src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -297,7 +299,7 @@ class FileField extends React.Component {
                       <ProgressBar percent={this.state.progress} />
                       <button
                         type="button"
-                        className="va-button-link"
+                        className={cancelBtnClass}
```

Comment:
This will probably need to be changed into a secondary button without making it an option. I recently did the same thing with the delete button - https://github.com/department-of-veterans-affairs/vets-website/pull/17054/files#diff-be73ef140f22ca1d406a85b603d7b1a3f022dc0ed50550a651c258127b10424cR372 - did you discuss this with the design team?

Reply:
Some discussion started [here](https://dsva.slack.com/archives/CBU0KDSB1/p1621018211435300?thread_ts=1621017046.433200&cid=CBU0KDSB1). Will update with comments when next steps are figured out.

Reply:
:wave: Like we talked about in Slack, you're good to go on making this a button for all forms instead of an option in the `uiSchema`. :+1: I see no drawback, nor did anybody who I asked.

Reply:
PR updated to unconditionally change the file upload cancel button.